ocs_ci/utility/utils.py

```diff
@@ -852,7 +852,10 @@ def email_reports():
     [recipients.append(mailid) for mailid in mailids.split(",")]
     sender = "ocs-ci@redhat.com"
     msg = MIMEMultipart('alternative')
-    msg['Subject'] = f"ocs-ci results for {get_testrun_name()} (RUN ID: {config.RUN['run_id']})"
+    msg['Subject'] = (
+        f"ocs-ci results for {get_testrun_name()} - {config.ENV_DATA.get('platform').upper()}-"
+        f"{config.ENV_DATA.get('deployment_type').upper()} (RUN ID: {config.RUN['run_id']})"
+    )
```

Comment:
I think, that this should be updated in the [`get_testrun_name()`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L1048), so it will be reflected also in the Polarion report, html report and so on.

Reply:
Done